lib/school_house_web/templates/lesson/lesson.html.eex

```diff
@@ -13,10 +13,67 @@
   </div>
   <%= raw @lesson.body %>
 
-  <blockquote class="edit-lesson">    
+  <blockquote class="edit-lesson">
     Caught a mistake or want to contribute to the lesson?
     <a href="https://github.com/elixirschool/elixirschool/edit/master/<%= @lesson.locale %>/lessons/<%= @lesson.section %>/<%= @lesson.name %>.md" target="_blank" rel="noopener">
       Edit this lesson on GitHub!
     </a>
   </blockquote>
 </section>
+
+<section class="bg-brand-light-gray -mb-16 mt-16 w-screen relative left-2/4 right-2/4 ml-half-screen mr-half-screen">
+  <div class="max-w-7xl mx-auto py-8 px-4 sm:px-6 lg:py-12 lg:px-8 items-center">
+      <div class="flex md:justify-between flex-col-reverse md:flex-row">
+        <div class="md:w-1/3 mt-8 md:mt-0">
+          <%= if @lesson.previous do %>
+            <div class="flex md:justify-start justify-center items-center content-center mb-6">
+              <svg xmlns="http://www.w3.org/2000/svg" class="text-gray-400 h-6 w-6 mr-2" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M15 19l-7-7 7-7" />
+              </svg>
+              <h3 class="leading-tight text-sm font-semibold text-gray-400 tracking-wider uppercase">
+                <a href="/<%= @lesson.previous.locale %>/<%= @lesson.previous.section %>/<%= @lesson.previous.name %>" class="hover:text-gray-900" rel="prev">
+                  Previous lesson
+                </a>
+              </h3>
+            </div>
+            <div class="text-center">
+              <p class="text-base font-semibold text-gray-900 mb-2">
+                <%= @lesson.previous.title %>
+              </p>
+              <p class="md:text-left text-center text-sm text-gray-500 overflow-hidden max-h-16">
+                <%= raw @lesson.previous.excerpt %>
+              </p>
+            </div>
+          <% end %>
+        </div>
+        <div class="md:w-1/3">
+          <%= if @lesson.next do %>
+            <div class="flex md:justify-end justify-center items-center content-center mb-6">
+              <h3 class="leading-tight text-sm font-semibold text-gray-400 tracking-wider uppercase">
+                <a href="/<%= @lesson.next.locale %>/<%= @lesson.next.section %>/<%= @lesson.next.name %>" class="hover:text-gray-900" rel="next">
```

Comment:
Let's use a link helper for this. I think `lesson_link/4` in `html_helpers.ex` might do the trick:


```eex
<%= lesson_link(@conn, @lesson.next.section, @lesson.next.name, "hover:text-gray-900"), do: @lesson.next.title %>
```

Reply:
We also have some "brand" colors defined in `tailwind.config.js` that could potentially be better here than `text-gray-900` but that's not a big one.